rdrf/rdrf/models/definition/models.py

```diff
@@ -899,10 +899,10 @@ def is_abnormal(self, value):
             # extract each individual rules from abnormality_condition
             # ignore empty lines
             abnormality_condition_lines = list(
-                filter(None, map(lambda rule: rule.strip(), self.abnormality_condition.split("\\r\\n")))
+                filter(None, map(lambda rule: rule.strip(), self.abnormality_condition.split("\r\n")))
```

Comment:
List comprehensions often look cleaner in code :  e.g.  lines = [ rule.strip() for rule in self.abnormality_condition.splitlines() ]